Cython/Compiler/ModuleNode.py

```diff
@@ -1106,6 +1106,34 @@ def generate_enum_definition(self, entry, code):
             # Not pre-declared.
             code.putln("typedef enum %s %s;" % (name, name))
 
+    def generate_scoped_enum_definition(self, entry, code):
```

Comment:
Hmm - it's totally possible to get rid of this as well. Looking into this.

Reply:
Was able to remove `generate_scoped_enum_definition` entirely and just extend `generat_enum_definition` to incorporate scoped enums